src/components/tx/modals/NewTxModal/ReplacementModal.tsx

```diff
@@ -0,0 +1,124 @@
+import CheckIcon from '@mui/icons-material/Check'
+import DeleteIcon from '@/public/images/common/delete.svg'
+import { SendTokensButton, SendNFTsButton } from './TxButton'
+
+import css from './styles.module.css'
+
+const wrapIcon = (icon: React.ReactNode) => <div className={css.circle}>{icon}</div>
+
+const steps = [
+  {
+    label: 'Create new transaction with same nonce',
+    icon: <div className={css.redCircle} />,
+  },
+  {
+    label: 'Collect confirmations from owners',
+    icon: wrapIcon(<CheckIcon fontSize="small" color="border" />),
+  },
+  {
+    label: 'Execute replacement transaction',
+    icon: wrapIcon(<SvgIcon component={RocketIcon} inheritViewBox fontSize="small" color="border" />),
+  },
+  {
+    label: 'The initial transaction is replaced',
+    icon: wrapIcon(<SvgIcon component={DeleteIcon} inheritViewBox fontSize="small" color="border" />),
+  },
+]
+
+const ReplacementModal = ({
+  txNonce,
+  onClose,
+  onTokenModalOpen,
+  onNFTModalOpen,
+  onRejectModalOpen,
+}: {
+  txNonce: number
+  onClose: () => void
+  onTokenModalOpen: () => void
+  onNFTModalOpen: () => void
+  onRejectModalOpen: () => void
+}) => {
+  return (
+    <ModalDialog open dialogTitle={`Replace transaction with nonce ${txNonce}`} onClose={onClose}>
+      <DialogContent>
+        <Box className={css.container}>
+          <Typography variant="h5" gutterBottom>
+            How to replace/change a transaction?
+          </Typography>
+          <Typography variant="body1" textAlign="center">
+            A signed transaction cannot be removed but it can be replaced with a new transaction with the same nonce.
+          </Typography>
+          <Stepper alternativeLabel className={css.stepper}>
+            {steps.map(({ label }) => (
+              <Step key={label}>
+                <StepLabel StepIconComponent={({ icon }) => steps[Number(icon) - 1].icon}>
+                  <Typography variant="body1" fontWeight={700}>
+                    {label}
+                  </Typography>
+                </StepLabel>
+              </Step>
+            ))}
+          </Stepper>
+        </Box>
+      </DialogContent>
+      <DialogActions>
+        <Grid container alignItems="center" justifyContent="center" spacing={2}>
+          <Grid item xs={12}>
+            <Typography variant="body2" textAlign="center" fontWeight={700}>
+              Select the type of replacement transaction
+            </Typography>
+          </Grid>
+          <Grid item xs>
+            <SendTokensButton onClick={onTokenModalOpen} sx={{ mb: 1 }} />
+            <SendNFTsButton onClick={onNFTModalOpen} />
+          </Grid>
+          <Grid xs={1}>
+            <Typography variant="body2" textAlign="center" fontWeight={700}>
+              or
+            </Typography>
+          </Grid>
+          <Grid xs>
+            <Button onClick={onRejectModalOpen} variant="outlined" fullWidth sx={{ mt: 2 }}>
+              Replace transaction
```

Comment:
IMHO this wording here makes the whole modal confusing. If this is this button replaces the tx, what do the two others do?

I would write "Empty transaction", "Stub transaction", "Rejection transaction" or something similar.